### ISSUE TITLE
Issue with sending chunks to client in certain worlds

### ISSUE THREAD

**Issue by erik1988** [PaperMC/Paper#207]
# Incompatibility (Bug)

http://postimg.org/image/uicbozpjr/

This picture illustrates whats happening. It seems to only be in the nether, but chunks are not visible unless you place a block down or relogg. I installed paperspigot yesterday and have not had this problem with normal spigot before so im pretty sure its caused by paper.  I tested to use normal spigot again and the problem disappeared. 

This is my config (and plugins):
http://timings.aikar.co/v2/?id=a900a03c9cad4c30ac644c4f53904050&section=config
### Paper build number:

git-Paper-676

**Comment by Techcable** [comment]
I had this issue in the overworld too.

**Comment by Spottedleaf** [comment]
I'm pretty sure @aikar has this issue as well.

> [13/04/2016  19:59:09  -0700]Aikar:   do yall have any idea whats corrupting my chunk sends to client?
> [13/04/2016  20:00:58  -0700]Aikar:   having horrible issues with client missing chunks
> [13/04/2016  20:01:00  -0700]Aikar:   having to relogin
> [13/04/2016  20:05:18  -0700]Aikar:   Spottedleaf: http://i.imgur.com/BQ6d8ih.png

Try dumping the packets your client receives.

**Comment by aikar** [comment]
here is entire diff of paper minus timings v2.... 
https://gist.github.com/aikar/b5e16454dbc952e8a74fdc37fa80d38a

If someone wants to review what might be causing it.

**Comment by aikar** [comment]
It seems to vary. I have it in my Town world, but not overworld/wastelands... I Reviewed entire diff of Paper and can't see any probable cause.

**Comment by Brokkonaut** [comment]
Maybe it is not only a Paper bug, but a Spigot bug: https://hub.spigotmc.org/jira/browse/SPIGOT-1760

**Comment by erik1988** [comment]
For me it only seems to be a problem when im using paper. I tried it again on a newly generated nether on git-Paper-690 (MC: 1.9.2). Overworld seems to work okay for me, but Its only a test server so its hard to tell. 

I had the same problems as described above, however when I tried the same world again in spigot (without changing anything else) everything works as expected.

as of now this is the only thing keeping me from using Paper instead of spigot.
